Feature/advance docs (#35)
infinum/eightshift-docs: docs/advanced/aliases.md

```js
import 'EightshiftBlocksSwiper';
```

## EightshiftBlocksSwiperIE

Alias providing [Swiper](https://www.npmjs.com/package/swiper) but for IE11 version.

```js
import 'EightshiftBlocksSwiperIE';
```

## EightshiftBlocksBabelPolyfill

Alias providing [Babel Polyfill](https://babeljs.io/docs/en/babel-polyfill).

```js
import 'EightshiftBlocksBabelPolyfill';
```

## EightshiftBlocksAutoprefixer

Alias providing [Autoprefixer](https://github.com/postcss/autoprefixer).

```js
import 'EightshiftBlocksAutoprefixer';
```

## EightshiftBlocksNormalize

Alias providing [Normalize](https://www.npmjs.com/package/normalize-scss). Check documentation for more details.

```scss
@import 'EightshiftBlocksNormalize';
@include normalize(); // stylelint-disable-line at-rule-empty-line-before
```

## EightshiftBlocksMediaBlender

Alias providing [Media Blender](https://github.com/infinum/media-blender). Check documentation for more details.

```scss
@import 'EightshiftBlocksMediaBlender';
```

## EightshiftBlocksSwiperStyle

Alias providing [Swiper](https://www.npmjs.com/package/swiper).

```scss
@import 'EightshiftBlocksSwiperStyle';

> ```

Reviewer: Should this be under the swiper one? Just to have similar things in similar places.

Author: I wanted to have the list the same order as it is in the libs so not to get confused.

I would like it like this for now and will move it once I fix this in the libs

## EightshiftBlocksDynamicImport

> To get (require) all the files using `require.context`  method, we created a helper located [here](https://github.com/infinum/eightshift-frontend-libs/blob/develop/scripts/dynamic-import.js).

Reviewer: ```suggestion
To get (require) all the files using the `require.context` method, we created a helper located [here](https://github.com/infinum/eightshift-frontend-libs/blob/develop/scripts/dynamic-import.js).
```

> It finds all files recursively in the folder using regex. This example will require all `assets/index.js` files inside the `custom` folder, so there is no need to add the files to the build manually.

Reviewer: ```suggestion
It will find all files recursively in the folder using a regex. The following example will require all `assets/index.js` files inside the `custom` folder, so there is no need to add the files to the build manually.
```

```js
import { dynamicImport } from 'EightshiftBlocksDynamicImport';

dynamicImport(require.context('./../../custom', true, /assets\/index.js$/));
```

## EightshiftBlocksRegisterBlocks

Alias providing blocks registration. For more details check [here](/eightshift-docs/docs/guides/blocks-registration).

## EightshiftBlocksUcfirst

> Converts the first letter of the string to uppercase.

Reviewer: ```suggestion
Converts the first letter of a string to uppercase.
```

```js
import { ucfirst } from 'EightshiftBlocksUcfirst';

ucfirst('custom string');
```

## EightshiftBlocksGetActions

Alias providing blocks actions helper. For more details check [here](/eightshift-docs/docs/guides/blocks-get-actions-helper).

## EightshiftBlocksUtilityHelpersPath

> We have also created a bunch of JavaScript helpers located in [helpers folder](https://github.com/infinum/eightshift-frontend-libs/tree/develop/scripts/helpers).

Reviewer: ```suggestion
We have also created a bunch of JavaScript helpers located in the [helpers folder](https://github.com/infinum/eightshift-frontend-libs/tree/develop/scripts/helpers).
```

> This alias provides only path to helpers folder.

Reviewer: ```suggestion
This alias provides the only path to the helpers folder.
```

What was the intention with this sentence? I'm not sure I get it 100%
